lib/renderer/api/remote.js

```diff
@@ -175,6 +175,7 @@ const proxyFunctionProperties = function (remoteMemberFunction, metaId, name) {
       return true
     },
     get: (target, property, receiver) => {
+      if (property === 'toString') return target.toString.bind(target)
```

Comment:
I believe that's redundant as a proxy can only receive a function as a parameter. If this is a safety check for `Proxy's` definition supporting more than functions, then it might be better to use an `assert` instead so we can detect the new case via error reporting and implement its behavior accurately instead of trying to guess:

```js
if (property === 'toString') {
  // Sanity check that Proxy only supports functions
  assert(target.toString);

  // Return our original toString
  return target.toString.bind(target);
}
```

Thoughts?